### PR TITLE
Add Member Management Panel

### LINKED ISSUE #28: User Story - Member Management (Project Profile)
# Story Description

As a project owner, I should be able to invite additional members from the project profile (members tab). I should also be able to remove existing members (to free up space if necessary).

# Acceptance Criteria

- UI on the "Members" tab of the project profile for inviting additional members.
- Button beside each user in the "Members" tab for removal.

# Additional Info

N/A

### PR DESCRIPTION
**Pull request by miosenpai**
This PR adds a member management panel to the project profile page. The member management panel allows the owner of a project to invite additional members & remove existing members.

Merge #32 before this.

Resolves #28.

### CODE REVIEW

**Review by TianYueDong — APPROVED** [review]
Looks good!

File: Buddies.API/Controllers/ProjectsController.cs [hunk 1]
```diff
@@ -404,8 +404,8 @@ public async Task<ActionResult> InviteProjectMember(int pid, int uid)
                 return NotFound("PROFILE NOT FOUND");
             }
             if (project.Owner != _userManager.GetUserAsync(User).Result) { return Unauthorized(); }
-
-            var invitedUser = _context.Users.FindAsync(uid).Result;
+
```
Comment (andy9775) [inline Buddies.API/Controllers/ProjectsController.cs]:
```suggestion

```

File: Buddies.Web/src/components/ProjectBuddies.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  MemberLst,
+  isOwner,
+  ownerId,
+  getUsers,
+  submitInvite,
+  InvitedLst,
+  submitRemoval,
+  isFull,
+}) => {
+  const [openedDialog, setOpenedDialog] = useState<Dialogs>('');
```
Comment (andy9775) [inline Buddies.Web/src/components/ProjectBuddies.tsx]: I think `dialog` is fine here. openedDialog implies it's already open

File: Buddies.Web/src/components/ProjectBuddies.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+  const [toRemove, setToRemove] = useState<UserInfo | null>(null);
```
Comment (andy9775) [inline Buddies.Web/src/components/ProjectBuddies.tsx]: `userToRemove`?

**Review by andy9775 — APPROVED** [review]
I would rename utils.ts to something else. having a utils file/folder is usually a bad sign. Also a few nits, otherwise LGTM!

**Pull request merged**